CandyShopper

ayaangrover opened the pull request:
# Your checklist for this pull request

## Author name

**Author: Ayaan Grover**

<!-- A name or nickname that you want to appear as the author of the game -->

## About your game

**What is your game about?**

Pushing candy into the shopping basket.

**How do you play your game?**

Press WASD to move, J to restart.

## Code
Check off the items that are true. <!-- Put a `x` in the  `[ ]`. Example: `[x] The game was made...` -->
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. <!-- Example: `sokoban_plus.js` -> `sokoban_plus.png`. -->

> Thanks for your PR!

GalaxyGamingBoy commented:
I really love the idea of this game but at its current state it can use one more game mechanic as to differentiate it from other sokoban based games, like a buff/de-buff depending on the item or even multiple players.

ayaangrover commented:
Hello @GalaxyGamingBoy, 

As per your suggestions, I have increased the number of levels from ~10 to 30, added a money sprite which gives you extra lives, added a lives counter, and designed certain levels so they require multiple lives to pass! Please take a look at it and let me know what you think.

Thanks,

Ayaan
